db/migrate/20150328144811_consolidate_plant_line_details.rb

```diff
@@ -0,0 +1,195 @@
+class ConsolidatePlantLineDetails < ActiveRecord::Migration
+  def up
+
+    # Rename existing PV to avoid naming clash
+    ActiveRecord::Base.connection.execute("ALTER TABLE plant_varieties \
+      RENAME TO old_plant_varieties")
+
+    create_table :plant_varieties do |t|
+      t.string :plant_variety_name,         primary: true
+      t.string :crop_type
+      t.string :comments
+      t.text :entered_by_whom,              default: "unspecified", null: false
+      t.date :date_entered
+      t.string :data_provenance
+
+      t.string :data_attribution,           default: "unspecified", null: false
+      t.string :country_of_origin,          default: "xxx", null: false
+      t.string :country_registered,         default: "xxx", null: false
+      t.string :year_registered,            default: "xxxx", null: false
+      t.string :breeders_variety_code
+      t.string :owner
+      t.string :quoted_parentage
+      t.string :female_parent
+      t.string :male_parent
+      t.string :detail_comments
+      t.string :detail_entered_by_whom,     default: "unspecified", null: false
+      t.date :detail_date_entered
+      t.string :detail_data_provenance
```

Comment:
Since we merged two arrays I don't think keeping these detail_x fields is useful. Can we make a sensible "merge" of these values with the values from original PV fields?